build.fsx

```diff
@@ -0,0 +1,95 @@
+#I "packages/build/fake/tools/"
+#r "FakeLib"
+
+open Fake
+open Fake.AssemblyInfoFile
+open Fake.Testing
+
+module AppVeyor = 
+    let BuildNumber = environVarOrNone "APPVEYOR_BUILD_NUMBER"
+
+open AppVeyor
+open Fake.Testing.NUnit3
+
+module ReleaseNotes = 
+    let TickSpec = ReleaseNotesHelper.LoadReleaseNotes ("./RELEASE_NOTES.md")
+
+module Build = 
+    let rootDir = __SOURCE_DIRECTORY__
+    let nuget = rootDir </> "packed_nugets"
+    let setParams defaults =
+        { defaults with
+            Verbosity = Some MSBuildVerbosity.Normal
+            Targets = ["Rebuild"]
+            Properties =
+                [ "AllowedReferenceRelatedFileExtensions", ".pdb"
+                  "Optimize", "True"
+                  "DebugSymbols", "True"
+                  "Configuration", "Release" ]
+         }
+
+let Sln = "./TickSpec.sln"
+
+Target "Clean" <| fun _ -> DeleteDir Build.nuget
+    
+Target "AssemblyInfo" <| fun _ ->
+    !! ("TickSpec" </> "AssemblyInfo.fs")
+    |> Seq.iter(fun asmInfo ->
+        let fileVersion =
+            BuildNumber |> Option.defaultValue "0" 
+            |> sprintf "%s.%s" ReleaseNotes.TickSpec.AssemblyVersion
+        [ Attribute.Version fileVersion
+          Attribute.FileVersion fileVersion ]
+        |> AssemblyInfoFile.UpdateAttributes asmInfo)
+
+Target "Build" <| fun _ -> build Build.setParams Sln
+
+Target "Test" <| fun _ ->
+    let examples x = Build.rootDir </> "Examples" </> x
```

Comment:
Is it possible to run the tests without listing all the assemblies here? It will be easy to miss it when adding new tests.

Reply:
Yes fixed into the last commit